docs/userguides/accounts.md

```diff
@@ -57,6 +57,21 @@ If your testing provider supports this feature, it is possible to directly set t
 account.balance += int(1e18)  # Gives `account` 1 Ether
 ```
 
+### Default Sender Support
+
+In order to eliminate the usage of sender in contract calls, you can use `use_sender` context manager.
+
+```python
+with test_accounts.use_sender(0):
```

Comment:
`test_accounts` is the `accounts` fixture in the context of running tests.
It make more sense to show that:

```python
def test_example(accounts):
    with accounts.use_sender(0):  # First account from test mnemonic
        ...
```

Reply:
but tests are not failing and if accounts is really accounts then __getitem_int wont also fallback to the test_accounts and this would even raise an error. couldnt get really

Reply:
we don't test the docs right now, so they wouldn't be failing